fieldservice_route_stock/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2019 Open Source Integrators
+# Copyright (C) 2019 Serpent Consulting Services
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Field Service Route Stock',
+    'version': '12.0.1.0.0',
+    'category': 'Field Service',
+    'author': 'Open Source Integrators, Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/field-service',
+    'depends': [
+        'account',
```

Comment:
```suggestion
```

Reply:
for Jornal entries, we need to add `account` dependency.